packages/api/src/auth.js

```diff
@@ -164,19 +170,19 @@ function authenticateMagicToken (
     return dangerousAuthBypassForUnitTesting.defaults
   }
 
-  /** @type {import('@magic-sdk/admin').ParsedDIDToken} */
+  /** @type {import('@magic-sdk/admin').ParsedDIDToken | null} */
   let decodedToken = null
 
   try {
     decodedToken = env.magic.token.decode(token)
   } catch (error) {
-    console.warn('error decoding magic token', error.name, error.message)
+    console.warn('error decoding magic token', error)
```

Comment:
minor but this will change how errors get logged... but maybe this is better tho

```console
❯ node
Welcome to Node.js v18.8.0.
Type ".help" for more information.
> console.warn('foo', 'bar', 'baz')
foo bar baz
undefined
> console.warn('foo', {name: 'bar', message: 'baz' } )
foo { name: 'bar', message: 'baz' }
undefined
> console.warn('foo', new Error('baz') )
foo Error: baz
    at REPL2:1:21
    at Script.runInThisContext (node:vm:129:12)
    at REPLServer.defaultEval (node:repl:572:29)
    at bound (node:domain:433:15)
    at REPLServer.runBound [as eval] (node:domain:444:12)
    at REPLServer.onLine (node:repl:902:10)
    at REPLServer.emit (node:events:525:35)
    at REPLServer.emit (node:domain:489:12)
    at [_onLine] [as _onLine] (node:internal/readline/interface:425:12)
    at [_line] [as _line] (node:internal/readline/interface:886:18)
undefined
```

Reply:
@olizilla I thought it would be fine... technically something could be thrown without a `.name` or `.message`, but with some other `toString()`, which I think is why tsc complains that `error` is `unknown` unless you do an `error instanceof Error` check.